docs/what.rst

```diff
@@ -0,0 +1,23 @@
+=====================
+What is a Certificate?
+=====================
+
+An SSL *certificate* contains a public key and a private key that work together to enable
+a web browser to communicate with your server securely over an encrypted SSL (secure socket layer)
+connection. The certificate also identifies your website so that the browser (and the person behind it)
+can trust your identity. SSL certificates have mainly been used to provide secure,
```

Comment:
Yes, the generic "certificate" info I got was from the web, and I knew that it might not be exactly the way you want to present them—thence the phrase "first draft" in the commit message. I figured we'd need to iterate. 

So, from what you're saying, the main points of the cert are to enable the encryption to happen (with the public/private keys) and to ensure that you're connecting to the site that actually has the cert associated with it (though there's no guarantee that the site is run by nice people).  Is that closer to the truth? I want to state the benefits accurately.